tabpfn_client/prompt_agent.py

```diff
@@ -115,3 +126,24 @@ def prompt_confirm_password_for_user_account_deletion(cls) -> str:
     @classmethod
     def prompt_account_deleted(cls):
         print(cls.indent("Your account has been deleted."))
+
+    @classmethod
+    def _choice_with_retries(cls, prompt: str, choices: list, max_attempts: int = 3) -> str:
```

Comment:
why have a max attempts? I think this could also go for ever, no?

Reply:
Yeah, you're right, I changed it. But not sure if it will ever make a difference :grinning:

Reply:
nah me neither, but it is one less trajectory in the code to think about. Simpler :)